Dockerfile

```diff
@@ -0,0 +1,34 @@
+# BASE IMAGE
+FROM mariadb:10.5.8
+
+# Me
+MAINTAINER gramadin@databygram.com
+
+# create and use folder for the image
+
+WORKDIR .

```

Comment:
I think this downloads everything in your pwd did you need everything?

Reply:
🤷‍♂️

Reply:
@gramadin As @dafrenchyman mentioned using the mariadb as the base image seems super cool. I think I might use that idea ;)